galley/pkg/config/processor/transforms/ingress/annotations/annotations.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotations
+
+import "istio.io/pkg/annotations"
+
+// IngressClass is the annotation on Ingress resources for the class of controllers responsible for it
+var IngressClass = annotations.Register(
```

Comment:
Nathan is working on PR that changes how annotations are handled, you might want to sync.

Reply:
Yeah, annotations have already been moved to istio/api.  I have a follow-on (#15520) which will remove them from istio/istio.